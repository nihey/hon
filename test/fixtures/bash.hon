echo "Nothing will stand in our way"

# I will finish what you started
touch __hon__anything__.txt
cat __hon__anything__.txt >> __hon__file.log
rm __hon__file.log __hon__anything__.txt
cd /tmp
echo $1
exit 1
