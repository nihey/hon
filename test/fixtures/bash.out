Nothing will stand in our way

